Reyes_JohnRuiz/index.html

```diff
@@ -159,26 +163,27 @@ <h2>Thoughts about online class</h2>
       </div>
     </main>
 
-    <div class="Contact">
+    <div class="contact">
       <h2>Contact Me</h2>
-      <form id="contact-form" class="form">
+      <form id="contact_form" class="form">
         <input type="email" name="email" id="email" placeholder="Email" />
         <input type="text" name="name" id="name" placeholder="Full Name" />
         <input type="text" name="subject" id="subject" placeholder="Subject" />
-        <input type="number" name="age" id="age" placeholder="Age"/>
+        <input type="number" name="age" id="age" placeholder="Age" />
         <label for="copy">Send me a copy</label>
         <input type="checkbox" name="copy" id="copy" />
         <textarea
           id="message"
           name="message"
           rows="3"
           cols="15"
-          placeholder="Message">
+          placeholder="Message"
+        >
         </textarea>
```

Comment:
Don't separate the closing tag in a new line.
```suggestion
          placeholder="Message">
        </textarea>
```